README.md

```diff
@@ -1,181 +1,34 @@
-# express-auth-template
-test
-A template for starting projects with `express` as an API. Includes
-authentication and common middlewares.
+##Project 3 - Art Gallery
+#Back End
 
-## Installation
+#Overview
 
-1. [Download](../../archive/master.zip) this template.
-1. Move the .zip file to your `sei/projects/` directory and Unzip it (creating a
-   folder) -- **NOTE:** if the folder was already unzipped, use the `mv` command
-   line to move it to the `sei/projects/` directory.
-1. Rename the directory from express-auth-template -> your-app-name.
-1. Rename [`README.md`](README.md) to `REF.md` and use as a reference, create a new README and fill with your own content.
-1. Move into the new project and `git init`.
-1. Replace all instances of `'express-auth-template'` with your app name.
-1. Install dependencies with `npm install`.
-1. Ensure that you have `nodemon` installed by running `npm install -g nodemon`.
-2. Once everything is working, make an initial commit.
+# User Stories 
 
-## Structure
+MVP (V1)
+As a user, I want the ability to….
+  -  Sign up
+  -  Sign in
+  -  Change my password
+  -  Sign out
+  -  Create my own gallery of artwork
+  -  View my gallery of artwork
+  -  Update the details of my own gallery
+  -  Delete artwork from my gallery
+  -  View galleries others have created
+  -  View an index of all of the galleries of artwork
 
-Dependencies are stored in [`package.json`](package.json).
+Bonus (V2)
 
-The most important file for understanding the structure of the template is
-`server.js`. This is where the actual Express `app` object is created, where
-the middlewares and routes are registered, and more. To register a routefile,
-follow the pattern established here with `exampleRoutes` and `userRoutes`. If
-you want to add any middlewares to your app, do that here.
+  -  Comment on artwork in other galleries
```

Comment:
I think there are a lot of possible paths included here for V2 and it would be helpful to narrow, outline exactly how we would build out just one. So perhaps either comments or search or auction or inquiries. My vote is for auction!

Reply:
totally fine with just choosing one path for now. if we get that done, we can add more later. np there at all. i also like the bidding idea, even if its only semi functional to simulate bidding, being outbid, and winning or losing.